src/main/java/dev/slimevr/config/FilteringConfig.java

```diff
@@ -0,0 +1,86 @@
+package dev.slimevr.config;
+
+import dev.slimevr.Main;
+import dev.slimevr.vr.trackers.IMUTracker;
+import dev.slimevr.vr.trackers.Tracker;
+import dev.slimevr.vr.trackers.TrackerFilters;
+import io.eiren.yaml.YamlNode;
+
+import java.util.HashMap;
+
+
+public class FilteringConfig {
+
+	private final ConfigManager configManager;
+
+	public static String CONFIG_ROOT = "filters";
+	public static float DEFAULT_INTENSITY = 0.3f;
+	public static int DEFAULT_TICK = 1;
+
+	private String type;
+	private float amount;
+	private int ticks;
+
+	private YamlNode rootNode;
+
+
+	public FilteringConfig(ConfigManager configManager) {
+		this.configManager = configManager;
+	}
+
+	public void loadConfig() {
+		this.rootNode = this.configManager.getConfig().getNode(CONFIG_ROOT);
+		if (rootNode == null)
+			this.rootNode = new YamlNode(new HashMap<>());
+		configManager.getConfig().setProperty(CONFIG_ROOT, this.rootNode);
+
+		this.type = this.rootNode
+			.getString("type", TrackerFilters.NONE.name());
+		this.amount = this.rootNode
+			.getFloat("amount", DEFAULT_INTENSITY);
+		this.ticks = this.rootNode
+			.getInt("tickCount", DEFAULT_TICK);
+	}
+
+	public void updateTrackersFilters(TrackerFilters filter, float amount, int ticks) {
+		setType(filter.name());
+		setAmount(amount);
+		setTicks(ticks);
+
+		IMUTracker imu;
+		for (Tracker t : Main.vrServer.getAllTrackers()) {
```

Comment:
This is some weird class dependency, unsure if this can be resolved, but it's not the best IMO

Reply:
Wdym by wierd class dependency

Reply:
Depending on a static insurance of the VR server when everything else takes in an instance of it, but I guess it's fine